wasm/jit/jit_amd64_test.go

```diff
@@ -1342,17 +1336,267 @@ func TestAmd64Builder_handleLe(t *testing.T) {
 				)
 				// Check the stack.
 				require.Equal(t, uint64(1), eng.stackPointer)
-				if tc.exp {
-					require.Equal(t, uint64(1), eng.stack[eng.stackPointer-1])
+				require.Equal(t, tc.x1 <= tc.x2, eng.stack[eng.stackPointer-1] == 1)
+			})
+		}
+	})
+}
+
+func TestAmd64Compiler_compileGe(t *testing.T) {
+	t.Run("int32", func(t *testing.T) {
+		for i, tc := range []struct {
+			x1, x2 int32
+			signed bool
+		}{
+			{x1: 100, x2: -1, signed: false},
+			{x1: -1, x2: -1, signed: false},
```

Comment:
a comment to help explain why we have signed false here
```suggestion
			{x1: -1, x2: -1, signed: false}, // interpret x1 and x2 as max uint32
```